models/intel/index.md

```diff
@@ -184,8 +184,7 @@ that predict an actual labels are called *decoder*.
 |---------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------- |----------- |
 | [driver-action-recognition-adas-0002-encoder](./driver-action-recognition-adas-0002-encoder/description/driver-action-recognition-adas-0002-encoder.md) | 0.676                | 2.863      |
 | [driver-action-recognition-adas-0002-decoder](./driver-action-recognition-adas-0002-decoder/description/driver-action-recognition-adas-0002-decoder.md) | 0.147                | 4.205      |
-| [action-recognition-0001-encoder](./action-recognition-0001-encoder/description/action-recognition-0001-encoder.md)                                     | 7.340                | 21.276     |
-| [action-recognition-0001-decoder](./action-recognition-0001-decoder/description/action-recognition-0001-decoder.md)                                     | 0.147                | 4.405      |
+| [action-recognition-0001](./action-recognition-0001/description/action-recognition-0001.md)                                     | 7.340                | 21.276     |
```

Comment:
Why not to specify sizes for both networks? Smth, like, 7.340 (encoder), 0.147 (decoder) | 27.276 (encoder), 4.405 (decoder)

Reply:
@snosov1 not sure we can do this in one line, so changed to two line variant with link point to the same description md file